src/app/components/global-styles/full-page-styles.tsx

```diff
@@ -10,7 +11,7 @@ export const fullPageStyles = css`
       max-height: unset;
       width: 100%;
     }
-    main.main-content {
+    .main-content {
       flex-direction: row;
```

Comment:
I saw this in many places and I am wondering why it's the case, `row` is the default value for flex direction, why is it set?

Reply:
I am not sure here, this pre-dates me.

Reply:
I removed this so I'm going to resolve this conversation.